src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/CreateMergedManifestBuildModelTests.cs

```diff
@@ -32,23 +29,81 @@ class CreateMergedManifestBuildModelTests
         private static readonly string buildNumber = "azDevBuildNumber";
         private static readonly string sourceBranch = "thisIsASourceBranch";
         private static readonly string commitSourceVersion = "thisIsASourceVersion";
+        private static readonly string id = "1234";
+        private static readonly string version = "thisIsVersion";
+        private static readonly string noVersion = "thisIsNoVersion";
 
-        private readonly AssetData assetDataWithoutName = new AssetData(true)
-        { Version = "noNameVersion" };
+        private PackageArtifactModel package1 = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id,
+            Version = version
+        };
+
+        private PackageArtifactModel package2 = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id ,
+            Version = noVersion
+        };
+
+        private PackageArtifactModel packageWithNoVersion = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id,
+            Version = null
+        };
+
+        private PackageArtifactModel packageNonShipping = new PackageArtifactModel
```

Comment:
This is called packageNonShipping, but you're setting NonShipping to false? Are these test objects inverted?

Reply:
Yes :( changed the name